includes/class-boldgrid-editor-assets.php

```diff
@@ -429,6 +433,32 @@ public function enqueue_drag_scripts() {
 			$this->get_js_vars()
 		);
 
+		/*
+		 * Localize wp-color-picker.
+		 *
+		 * This is only needed in WordPress >= 5.5 because wpColorPickerL10n has been removed.
+		 * @see https://github.com/WordPress/WordPress/commit/7e7b70cd1ae5772229abb769d0823411112c748b
+		 *
+		 * This is only needed until the wp-color-picker-alpha repo has been updated.
+		 * @see https://github.com/kallookoo/wp-color-picker-alpha/issues/35
+		 *
+		 * The preg_replace is needed to change 5.5-beta3-48571 to 5.5-3-48571 so that the version_compare
+		 * will work properly during the beta phase.
+		 */
+		if ( version_compare( preg_replace( "/[^0-9.-]/", "", $wp_version ), '5.5', '>=' ) ) {
```

Comment:
Could we do any version greater than 5.4.99

Reply:
Tested, looks like it works. Resolved via https://github.com/BoldGrid/post-and-page-builder/pull/141/commits/ad6c2a859c5c4de587da219f32089e986cd53e53